man/systemd.network.xml

```diff
@@ -1741,6 +1741,17 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>AssignAcquiredDelegatedPrefixAddress=</varname></term>
+          <listitem>
+            <para>Takes a boolean. In DHCPv6-PD environment, where WAN interface requests IPv6 via DHCPv6,
+            receives address as well as delegated prefixes, with LAN interfaces serving those delegated
+            prefixes in their router advertisement messages. When true, adds an address to LAN interface from
+            the delegated prefixes which are received from the WAN interface.
+            Default to false.</para>
```

Comment:
This should probably document that it uses EUI64.

IMHO it contains too much "noise". I think the whole story about that it carries the prefixes in the RA's shouldn't be there. It is a detail of another option that is in play here. I guess you could use this option and disable RA's on the given interface and this would still work. Right?



These three points are what I think should primarily in the manpage:
- Takes a boolean (defaults to false)
- If enabled assigns a EUI64 address from the DHCPv6-PD prefix
*on the "current" (as in that interface this .network unit matches)* to the interface.
- This option should commonly be enabled on the LAN interface of an DHCPv6-PD setup.

Reply:
I don#t think that at all a noise rather explains in detail .  Adding EUI64 is improvement.